### PR TITLE
Amazon timestream in PHP example app

### PR DESCRIPTION
**Pull request by dkovalenkoI**
## Summary
Create the started application Amazon Timestream with PHP

## Description
Provide a starter app for PHP to use the Amazon Timestream application and the installation guide.

## Related Issue
[awslabs#42](https://github.com/awslabs/amazon-timestream-tools/issues/42)

### Manual verification:
1. Install PHP based on the guide
2. Install Amazone SDK with Composer
3. Execute the application by `main.php`
4. Verify that the database, table and records have been created
5. Verify query results
6. Verify table and database are successfully deleted

## Additional Reviewers
@alexeytemnikov

### CODE REVIEW

**Comment by sethusrinivasan** [comment]
Thank you @dkovalenkoI

**Comment by mandawat** [comment]
Changes LGTM, can you resolve the conflicts?

File: sample_apps/php/main.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }
+
+    function deleteTable($client, $dbName, $dbTableName) {
+        return $client->deleteTable([
+                   'DatabaseName' => $dbName,
+                   'TableName' => $dbTableName,
+               ]);
+    }
+
+    function writeRecords($client, $dbName, $dbTableName, $records) {
```
Comment (mandawat) [inline sample_apps/php/main.php:65]: Can you add rejected records exception handling? Sample snippet provided below

```
if (isset($response['RejectedRecords']) && !empty($response['RejectedRecords'])) {
        echo "Some records were rejected:\n";
        foreach ($response['RejectedRecords'] as $rejectedRecord) {
            echo "Reason: " . $rejectedRecord['Reason'] . "\n";
            echo "ExistingRecord: " . json_encode($rejectedRecord['ExistingRecord']) . "\n";
        }
    } else {
        echo "Data inserted successfully!\n";
    }
```

Comment (dkovalenkoI) [reply sample_apps/php/main.php:65]: Sample has been added, thank you for your comment

**Review by mandawat — APPROVED** [review]

**Pull request merged**